2015-16/C01 Make Episode File.sps

```diff
@@ -369,7 +362,38 @@ aggregate
     /cij_ppa = Max(PPA).
 
 sort cases by chi keydate1_dateformat.
-
+<<<<<<< HEAD
```

Comment:
This shouldn't be here. Was there a merge conflict?

Reply:
Line removed